src/App.tsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
 
 import './App.scss';
+import { PostList } from './components/PostList/PostList';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts = posts.map(post => (
+  {
+    ...post,
+    user: users.find(user => user.id === post.userId) as User,
```

Comment:
`find` returns `User` or `undefined`, so there can be cases when user is undefined it's not okay